src/cli/cmd/login/mod.rs

```diff
@@ -63,15 +63,16 @@ impl LoginSubcommand {
         // $XDG_CONFIG_HOME/nix/nix.conf; basically ~/.config/nix/nix.conf
         let nix_config_path = xdg.place_config_file("nix/nix.conf")?;
         // $XDG_DATA_HOME/fh/netrc; basically ~/.local/share/flakehub/netrc
-        let netrc_path = xdg.place_data_file("flakehub/netrc")?;
+        let netrc_file_path = xdg.place_data_file("flakehub/netrc")?;
         // $XDG_CONFIG_HOME/fh/auth; basically ~/.config/fh/auth
         let token_path = auth_token_path()?;
 
-        let nix_config_addition = format!("\nnetrc-file = {}\n", netrc_path.display());
+        let netrc_file_string = netrc_file_path.display().to_string();
+        let nix_config_addition = format!("\nnetrc-file = {}\n", netrc_file_string);
         let netrc_contents = format!(
             "\
-            machine {frontend_host} login FIXME password {token}\n\
-            machine {backend_host} login FIXME password {token}\n\
+            machine {frontend_host} login flakehub password {token}\n\
+            machine {backend_host} login flakehub password {token}\n\
```

Comment:
In fact, we might should delegate this to a lib: https://docs.rs/netrc-rs/0.1.2/netrc_rs

Reply:
I rather like the current form, where the `flakehub` entries are right next to each other. Since this is a file "just for us" and shouldn't have any other entries in it, I don't think we need to optimize for readability, especially when the `password` is already quite long.

Reply:
All right-y, fair enough